pom.xml

```diff
@@ -233,6 +233,11 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <version>${maven-surefire-plugin.version}</version>
+        <configuration>
+          <excludes>
+            <exclude>**IT*</exclude>
```

Comment:
You might want to use JUnit5 instead: https://junit.org/junit5/docs/current/user-guide/#writing-tests-tagging-and-filtering

Reply:
Good point, I changed it.